src/main/java/edu/umich/its/cpm/MigrationTaskService.java

```diff
@@ -1594,15 +1596,15 @@ private String getHeaderAttribute(JSONArray headers, String attribute)
 		 * @return
 		 */
 
-		String  addEmailToGoogleGroup(String googleGroup, String rcf822Email) {
-			String ggb_server = env.getProperty(Utils.GGB_SERVER_NAME);
+		String addEmailToGoogleGroup(String googleGroup, String rcf822Email) {
 			log.info("addEmailToGoogleGroup: group: {}",googleGroup);
 			log.info("addEmailToGoogleGroup: email: {}",rcf822Email);
-			GGBApiWrapper ggb = new GGBApiWrapper(ggb_server,null);
+
+			GGBApiWrapper ggb = establishGGBConnection();
+
 			String archive_url = "/groups/"+googleGroup+"/messages";
-			String response = ggb.post_request(archive_url,rcf822Email);
-			// TODO: error handling
-			return response;
+			ApiResultWrapper arw= ggb.post_request(archive_url,rcf822Email);
+			return arw.getStatus().toString();
```

Comment:
So i like that you are sending the Status code, isn't it better to send the ApiResultWrapper instead since if !200 then the wrapper will have a Error response then i can put the whole error Json as string.